src/flags_menu.cpp

```diff
@@ -101,12 +110,11 @@ void FlagsMenu::render(Font& font) {
                 tp_gameInfo.have_sense_flag ^= 0x08;
                 break;
             }
-            
+
```

Comment:
Trailing whitespace was removed from this line, but the blank line itself could probably be removed.